src/client-services/github.client-service.ts

```diff
@@ -1,33 +1,38 @@
-import axios, { AxiosResponse, AxiosInstance } from 'axios';
+import axios, { AxiosResponse, AxiosInstance } from 'axios'
```

Comment:
Does removing the semi-colon cause lint errors?

After we update the linter, this should cause errors or be auto fixed.

Reply:
I think eslint auto removes semicolon. I tried putting at the end, but it got removed after saving the file :thinking:

Reply:
Interesting, after we update our Eslint rules, it should auto add semi-colons.

This is fine for now.